pilz_jog_arm_support/README.md

```diff
@@ -0,0 +1,47 @@
+# Jog Arm Support
+This Package contains files to launch the prbt with moveit_jog_arm and tork-a/fake_joint in simulation mode.
+
+## Prerequisites
+- You need a workstation or [Virtual machine](https://en.wikipedia.org/wiki/Virtual_machine) with [Ubuntu 18.04 LTS](http://releases.ubuntu.com/releases/18.04/) and [ROS melodic](http://wiki.ros.org/melodic) installed.
+- Create a [catkin workspace](http://wiki.ros.org/catkin/Tutorials/create_a_workspace)
+
+## Installation from source
+- clone this package into the src directory `git clone https://github.com/PilzDE/pilz_teach` 
+
+- Install moveit packages from source ...
+  - following the [moveit documentation](https://moveit.ros.org/install/source/)
+  - or with the packages `apt install ros-melodic-moveit` (currently jog_arm is not yet available as binary!)
```

Comment:
sounds like installing jog_arm from source is optional, maybe extend this line with:
or with the packages ... and only build jog_arm from source

Reply:
ah, that was your intention. I'll try to rephrase. I wanted to say "once jog_arm is released, you can simply install the binaries"...